test/parallel/test-child-process-spawn-typeerror.js

```diff
@@ -1,15 +1,28 @@
 'use strict';
-var assert = require('assert');
-var child_process = require('child_process');
-var spawn = child_process.spawn;
-var cmd = require('../common').isWindows ? 'rundll32' : 'ls';
-var invalidArgsMsg = /Incorrect value of args option/;
-var invalidOptionsMsg = /options argument must be an object/;
-
-// verify that args argument must be an array
-assert.throws(function() {
-  spawn(cmd, 'this is not an array');
-}, TypeError);
+var assert = require('assert'),
```

Comment:
Yes. I'd prefer that actually. Just literally pulled this over straight from what is in v0.12.